# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

DoHeadMoveActionGoal action_goal
DoHeadMoveActionResult action_result
DoHeadMoveActionFeedback action_feedback
